packages/core/src/types.ts

```diff
@@ -90,3 +90,5 @@ export type PickUnion<UnionType, Keys> = Exclude<
   UnionType,
   Exclude<UnionType, Keys>
 >;
+
+export type Simplify<T> = { [KeyType in keyof T]: T[KeyType] };
```

Comment:
Naming here is a bit confusing. Simplify what?

Reply:
It writes out the "end result" of combined types

```tsx
type ExternalUnion = 'a' | 'b'

type Local = Simplify<ExternalUnion | 'x' | 'y' | 'z'>

// Local #=> 'a' | 'b' | 'x' | 'y' | 'z'
```

Just taken from https://github.com/sindresorhus/type-fest/blob/main/source/simplify.d.ts

Reply:
Ok if it is an original name from type-fest